drivers/sensor/maxim/max31865/max31865.c

```diff
@@ -177,15 +177,21 @@ static char *max31865_error_to_string(uint8_t fault_register)
 static int max31865_fault_register(const struct device *dev)
 {
 	uint8_t fault_register;
+	uint8_t old_d5_d3_d2;
 
 	max31865_spi_read(dev, (REG_FAULT_STATUS), &fault_register, 1);
 	struct max31865_data *data = dev->data;
+	old_d5_d3_d2 = data->config_control_bits & BITMASK_D5_D3_D2;
 	/*Clear fault register */
 	WRITE_BIT(data->config_control_bits, 1, 1);
+	WRITE_BIT(data->config_control_bits, 5, 0);
```

Comment:
This could be done by applying the already defined mask instead. Would recommend naming the mask in a way that signals to the reader its for clearing fault bits

Reply:
I've renamed the mask to `FAULT_BITS_CLEAR_MASK ` to clearly indicate its purpose in clearing fault status bits.